src/ec/packages/ec-component-search-form/search-form.story.js

```diff
@@ -0,0 +1,38 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+
+import searchFormDocs from './docs/search-form.md';
+import searchForm from './search-form.html.twig';
+
+storiesOf('Components/SearchForm', module)
```

Comment:
that's a small thing, but display in storybook will be better if you add a space between Search and Form here (that's how the item is displayed in the menu)

Reply:
Do you mean [placeholder](https://www.w3schools.com/tags/att_input_placeholder.asp)?

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/28/commits/72483ba8d0c2d29290bdd9a64af43fa2ae0d59e7

Reply:
Please check if I added it well in https://github.com/ec-europa/ecl-twig/pull/28/commits/72483ba8d0c2d29290bdd9a64af43fa2ae0d59e7